SU-SOE/stanford_bean_types
# 7.x 2.x dev default image

#  READY FOR REVIEW

# Summary
See ticket: https://stanfordits.atlassian.net/browse/SOE-3185 
The stanford_bean_types feature is overridden on Prod. Namely, the default icon for the stanford_postcard_linked is in the DB and not in code. When we revert stanford_bean_types this will be lost.

This PR will put that icon into code and make it available as default to stanford_postcard_linked beans.

# Needed By (Date)
- Immediately

# Criticality
- Critical: Needed before release

# Steps to Test
- Switch to this branch: `git checkout 7.x-2.x-dev-default-image`
- Revert stanford_bean_types: `drush fr stanford_bean_types`
- Update database `drush updb`
- Navigate to the home page and verify that you see the icons on the images:
![screen shot 2018-06-22 at 23 16 47](https://user-images.githubusercontent.com/284440/41806421-81f3e310-7672-11e8-8766-fe3445fad10b.png)


# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
Work: https://stanfordits.atlassian.net/browse/SOE-3120
PR: https://stanfordits.atlassian.net/browse/SOE-3185

## Related PRs

## More Information

## Folks to notify
@kerri-augenstein


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
